src/library/Tabs/__tests__/TabList.spec.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+import React from 'react';
+import { shallow } from 'enzyme';
+import TabList, { componentTheme } from '../TabList';
+import testThemeOverrides from '../../../../utils/testThemeOverrides';
+import { mountInThemeProvider } from '../../../../utils/enzymeUtils';
+import { getProcessedComponentThemeKeys } from '../../themes/processComponentTheme';
+
+/*
+ * [1] This getter does not exist on the HTMLElement.prototype in JSDOM, so we
+ *     must mock it on the global. `clientWidth` and `offsetLeft`, also used by
+ *     TabList always return 0, which is conveniently useful.
+ * [2] Then we must clean up our global mocking in [1]
+ */
+
+function shallowTabList(props = {}) {
+  const tabListProps = {
+    onIncrement: handleIncrement,
+    ...props
+  };
+  return shallow(<TabList {...tabListProps} />);
+}
+
+let index = 0;
+function handleIncrement(direction: string, event: SyntheticEvent<*>) {
```

Comment:
Actually, `handleIncrement` does not appear to be used at all in this file.

Reply:
👍